start-console.sh

```diff
@@ -4,6 +4,10 @@ set -euo pipefail
 
 CONSOLE_IMAGE=${CONSOLE_IMAGE:="quay.io/openshift/origin-console:latest"}
 CONSOLE_PORT=${CONSOLE_PORT:=9000}
+CONSOLE_IMAGE_PLATFORM=${CONSOLE_IMAGE_PLATFORM:="linux/amd64"}
+
+# Plugin metadata is declared in package.json
+PLUGIN_NAME=${npm_package_consolePlugin_name}
```

Comment:
I am getting below error when I run the script

`./start-console.sh: line 10: npm_package_consolePlugin_name: unbound variable`